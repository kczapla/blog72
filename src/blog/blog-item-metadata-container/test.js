import React from 'react'
import renderer from 'react-test-renderer'

import BlogItemMetadataContainer from './index'

it('renders correctly', () => {
  const tree = renderer
    .create(<BlogItemMetadataContainer author={ { name: 'tmp', href: 'tmp' } } created={'2020-01-01'}/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
