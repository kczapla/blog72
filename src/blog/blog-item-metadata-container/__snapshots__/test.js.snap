// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdfBwQ iIIyjq"
>
  <time
    dateTime={"2020-01-01T00:00:00.000Z"}
  >
    Jan 01, 2020
  </time>
  <span
    className="sc-gsTCUz ra-dMI"
  >
     by 
  </span>
  <span
    className="sc-dlfnbm hahudT"
  >
    <a
      className="sc-hKgILt blFMmE"
      href="tmp"
    >
      tmp
    </a>
  </span>
</div>
`;
